### ISSUE TITLE
Documentation issues regarding Enterprise editions

### ISSUE THREAD

**Issue by maxb** [hashicorp/vault#18525]
I've spotted places where the Vault repository makes inconsistent statements about Enterprise feature levels:

Indicates Namespaces feature requires Enterprise Pro:

https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/ui/app/templates/vault/cluster/access/namespaces/index.hbs#L66
https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/ui/app/templates/vault/cluster/access/namespaces/create.hbs#L23

Indicates Namespaces feature available in all Enterprise versions:

https://github.com/hashicorp/vault/blob/main/website/content/docs/enterprise/namespaces.mdx#overview

----

Indicates Sentinel feature requires Enterprise Premium:

https://github.com/hashicorp/vault/blob/main/website/content/api-docs/system/policies.mdx

> ~> NOTE: This endpoint is only available in Vault version 0.9+. Please also note that RGPs and EGPs are Vault Enterprise Premium features and the associated endpoints are not available in Vault Open Source or Vault Enterprise Pro.

Indicates Sentinel feature requires Enterprise Plus:

https://github.com/hashicorp/vault/blob/main/website/content/docs/enterprise/sentinel/index.mdx

**Comment by maxb** [comment]
Found some more of these:

Indicates Control Groups feature requires Enterprise Premium:

https://github.com/hashicorp/vault/blob/159b60a181bae2d54c666ad09ca607988e4bf5c5/ui/app/templates/vault/cluster/access/control-group-accessor.hbs#L20
https://github.com/hashicorp/vault/blob/159b60a181bae2d54c666ad09ca607988e4bf5c5/ui/app/templates/vault/cluster/access/control-groups-configure.hbs#L11
https://github.com/hashicorp/vault/blob/159b60a181bae2d54c666ad09ca607988e4bf5c5/ui/app/templates/vault/cluster/access/control-groups.hbs#L38

Indicates Control Groups feature requires Enterprise Plus:

https://github.com/hashicorp/vault/blob/main/website/content/docs/enterprise/control-groups.mdx

Indicates Control Groups feature requires just Enterprise:

https://github.com/hashicorp/vault/blob/main/website/content/api-docs/system/config-control-group.mdx

**Comment by yhyakuna** [comment]
Good point... The pricing package has changed that we no longer have 'Premium' package. We'll go through them all and have a standard verbiage. Thx!

cc @schavis
